guibot/finder.py

```diff
@@ -114,19 +114,20 @@ def from_string(raw):
         :raises: :py:class:`ValueError` if unsupported type is encountered
         """
         args = []
-        string_args = re.match(r"<value='(.+)' min='([\d.None]+)' max='([\d.None]+)'"
+        string_args = re.match(r"<value='(.+)' min='(-?[\d.None]+)' max='([\d.None]+)'"
```

Comment:
I don't recall seeing any negative integers here for the currently implemented backends but in any case it is a useful generalization.

Reply:
Yep, there's an example [here](https://github.com/intra2net/guibot/blob/master/guibot/finder.py#L610)